apps/daimo-mobile/src/view/screen/InviteScreen.tsx

```diff
@@ -282,7 +282,7 @@ function ReferralButtonsFooter({
 
   const shareFarcaster = () => {
     console.log(`[INVITE] share on farcaster`);
-    const msg = encodeURIComponent(`Join me on Daimo`);
```

Comment:
Warpcast handles URL encoding weirdly. Both ways above should work equally. Instead, we se `Join%20me`... the first way.